server/mesh.go

```diff
@@ -61,45 +68,48 @@ func NewMeshHandler(loggerFactory LoggerFactory, wss *WSS, active_rooms map[stri
 					"userId": clientID,
 					"signal": signal,
 				}))
-/* 			case "ping":
+
+				/* 			case "ping":
 				log.Printf("Send pong message to %s ", clientID)
 				err = adapter.Emit(clientID, NewMessage(responseEventName, room, map[string]interface{}{
 					"message": "pong",
 				})) */
 			case "create_room":
 				payload, _ := msg.Payload.(map[string]interface{})
 				roomReq, _ := payload["room"].(string)
-				roomCreatorID, _ := payload["userId"].(string)
+
 				if roomExists(roomReq, active_rooms) {
 					err = adapter.Emit(clientID, NewMessage("room_created", room, map[string]interface{}{ //TODO: room?
 						"successful": "0",
 						"creatorId":  getRoomCreator(roomReq, active_rooms),
 					}))
 				} else {
-					createRoom(roomCreatorID, room, active_rooms)
+					createRoom(token["user_id"].(string), room, active_rooms)
 					err = adapter.Emit(clientID, NewMessage("room_created", room, map[string]interface{}{ //TODO: room?
 						"successful": "1",
-						"creatorId":  roomCreatorID,
+						"creatorId":  token["user_id"].(string),
 					}))
 				}
 
 			case "record":
+
 				payload, _ := msg.Payload.(map[string]interface{})
 				status, _ := payload["recordStatus"].(bool)
-				if clientID != getRoomCreator(room, active_rooms) {
-          err = adapter.Broadcast(
-                NewMessage("record_callback", room, map[string]interface{}{
-                  "successful": false,
-                }),
-              )
+
+				if token["user_id"].(string) != getRoomCreator(room, active_rooms) {
+					err = adapter.Broadcast(
+						NewMessage("record_callback", room, map[string]interface{}{
+							"successful": false,
+						}),
+					)
 				} else {
 					err = adapter.Broadcast(
-            NewMessage("record_callback", room, map[string]interface{}{
-              "successful": true,
-              "recordStatus": status,
-              "url": "ws://localhost:8882",
-            }),
-          )
+						NewMessage("record_callback", room, map[string]interface{}{
+							"successful":   true,
+							"recordStatus": status,
+							"url":          "ws://localhost:8882",
```

Comment:
[Minor] it was written by me, but probably it makes sense to export it to new config file